source/documentation/modules/map.html.md.erb

```diff
@@ -28,78 +67,297 @@ title: sass:map
     ===
     $font-weights: ("regular": 400, "medium": 500, "bold": 700)
 
-    @debug map.has-key($font-weights, "regular")  // true
-    @debug map.has-key($font-weights, "bolder")  // false
+    @debug map.has-key($font-weights, "regular") // true
+    @debug map.has-key($font-weights, "bolder") // false
+  <% end %>
+
```

Comment:
We usually separate different overloads of the same function with `---`, like you did for `map.set()`.